charts/atlantis/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: v0.22.3
+appVersion: v0.23.0
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.10.3
+version: 4.11.0
```

Comment:
Please also update the chart value

https://github.com/runatlantis/helm-charts/blob/a565c6a99be45d2bb9d973a791e2c3ca9fd2eaa1/charts/atlantis/values.yaml#L115

Please make it point to `latest` so it's one less thing we need to do when creating new releases. `latest` always points to the latest stable release.

Reply:
Hi @nitrocode @jamengual

it's not an good idea to use the latest tag. Kubernetes will never pull the new image with the latest tag if the POD starts on the same node again and if the pullPolicy is set to "IfNotPresent". Leave it empty as before and the appVersion will used. It's not best practice to use the latest tag for Helm Chart with semantic versioning. You don't have a fixed releases with a fixed appVersion/Container Image.

Please, revert this to have a stable and fixed release with a fixed images that relies to the corresponding chart Version.

https://vsupalov.com/docker-latest-tag/

Reply:
We can switch it to Always pull. This would allow latest and less maintenance for us.

The image tag is also overwritable, no?

For now, I put this PR in https://github.com/runatlantis/helm-charts/pull/266